packages/fxa-shared/metrics/glean/fxa-ui-metrics.yaml

```diff
@@ -57,6 +57,25 @@ event:
     data_sensitivity:
       - interaction
 
+  nimbus_user_id:
```

Comment:
Not part of this change, but this event along with the other three (name/reason/third_party_links) are meant to be extra keys in metric events, but the way our metrics are setup (see: `stringEventPropertyNames`) means that we have to generate these events even though we will never send them (!).